src/App/Fossa/Container/Sources/DockerEngine.hs

```diff
@@ -26,16 +26,17 @@ analyzeFromDockerEngine ::
   , Has Debug sig m
   ) =>
   Text ->
+  Text ->
   m ContainerScan
-analyzeFromDockerEngine imgTag = do
-  withSystemTempDir "fossa-docker-engine-tmp-" $ \dir -> do
+analyzeFromDockerEngine imgTag dockerHost = do
+  withSystemTempDir "fossa-docker-engine-tmp" $ \dir -> do
     let tempTarFile = dir </> $(mkRelFile "image.tar")
 
     logInfo . pretty $
       "Exporting docker image to temp file: "
         <> toString tempTarFile
         <> "! This may take a while!"
-    runDockerEngineApi $ getDockerImage imgTag tempTarFile
+    runDockerEngineApi dockerHost $ getDockerImage imgTag tempTarFile
 
     logInfo . pretty $ "Analyzing exported docker tarball: " <> toString tempTarFile
-    analyzeExportedTarball tempTarFile
+    analyzeFromDockerArchive tempTarFile
```

Comment:
If we're replacing "tarball" with "archive" in the types, we should probably do the same for logs/errors.